Homework2/StackCalculator/TestStackCalculator/TestStackCalculator.cs

```diff
@@ -0,0 +1,82 @@
+using NUnit.Framework;
+using StackCalculator;
+using System;
+
+namespace TestStackCalculator;
+
+/// <summary>
+/// A class for testing a stack calculator
+/// </summary>
+public class TestsStackCalculator
+{
+    Calculator? stackCalculator;
+    [SetUp]
+    public void Setup()
+    {
+       stackCalculator = new Calculator();
+    }
+
+    [Test]
+    public void DevideByZero()
```

Comment:
```suggestion
    public void DivideByZero()
```
Кстати, вот хорошая статья про именование юнит-тестов: https://dzone.com/articles/7-popular-unit-test-naming. Как у Вас --- плохо